mm2src/coins/lp_coins.rs

```diff
@@ -3177,15 +3178,22 @@ pub enum WithdrawError {
     },
     #[display(fmt = "Signing error {}", _0)]
     SigningError(String),
-    #[display(fmt = "Eth transaction type not supported")]
+    #[display(fmt = "Transaction type not supported")]
     TxTypeNotSupported,
-    #[display(fmt = "'chain_registry_name' was not found in coins configuration for '{}'", _0)]
-    RegistryNameIsMissing(String),
     #[display(
-        fmt = "IBC channel could not found for '{}' address. Consider providing it manually with 'ibc_source_channel' in the request.",
-        _0
+        fmt = "IBC channel could not found from coins file for '{}' address. Provide it manually by including `ibc_source_channel` in the request. See https://ibc.iobscan.io/channels for reference.",
```

Comment:
nit: grammatical fix `could not be found in coins file`. Also, should we refer the users to urls in errors, how secure is this approach?

Reply:
I am quite sure in the safety of their service and it could also be useful for GUI developers while they work on IBCs. But I can also delete it if you want, I am fine with doing that.

Reply:
Sure, [removed](https://github.com/KomodoPlatform/komodo-defi-framework/pull/2298/commits/b2add2fa567758ff3aa31a3be89b783ceef64beb).